try{
    setTimeout(function () {
        throw Error('xxx');
    })
}catch(e){

}